docs/Backend/02. jwt-oauth2-architecture.md

```
@@ -0,0 +1,898 @@
+    FE->>SB: GET /api/bookmarks (재시도)<br/>(Bearer: 새 AT)
+    SB-->>FE: { bookmarks: [...] }
+```
+
+**핵심 포인트:**
+- `fetchClient`가 **401을 자동 감지**하여 토큰 갱신 후 원래 요청을 재시도 (1회만)
+- 여러 API가 동시에 401을 받으면 `refreshPromise` 공유로 갱신 요청 1회만 발생
+- 갱신 실패(Refresh Token도 만료) → `authStore.logout()` → `/login` 이동
+
+> 💡 **백엔드 개발자를 위한 프론트엔드 로직 상세 (`fetchClient.ts`)**
+>
+> 1. **401 감지 (`response.status === 401`)**: 
```

markdownlint-cli2 (0.22.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)